examples/knative/go/kn-go-tagging/tagging/testdata/event.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
wondering what's the difference between these `event[n].json` events? They all look the same?

Reply:
I can probably get rid of event.json. As for the other two, `event2.json` uses` vm-60` and `event3.json` uses `vm-63`. I made those different so the tests can run in parallel.